cli/src/commands/feature-subgraph/index.ts

```diff
@@ -2,12 +2,14 @@ import { Command } from 'commander';
 import { BaseCommandOptions } from '../../core/types/types.js';
 import { checkAuth } from '../auth/utils.js';
 import CreateFeatureSubgraphCommand from './commands/create.js';
+import PublishFeatureSubgraphCommand from './commands/publish.js';
 
 export default (opts: BaseCommandOptions) => {
   const command = new Command('feature-subgraph').alias('fs');
   command.description('Provides commands for creating and managing feature subgraphs.');
```

Comment:
Can we add here a small definition of feature subgraphs?

Reply:
Done